README.md

```diff
@@ -1 +1,173 @@
+**Integration Authorization in Spring Boot API using AuthAction**
+This is a Spring Boot application demonstrating how to integrate API authorization using AuthAction with Spring Security OAuth2 Resource Server and JWKS for token validation.
+
+**Overview**
+This application showcases how to configure and handle authorization using AuthAction’s access token in a Spring Boot REST API.
+It uses JSON Web Tokens (JWT) for authentication and authorization and validates them dynamically using JWKS (JSON Web Key Sets) provided by AuthAction.
+
+**Prerequisites**
+Before using this application, ensure you have:
+
+Java 17+ installed
+
+Maven installed (for building and running the project)
+
+AuthAction API credentials:
+
+Tenant Domain
+
+API Identifier (Audience)
+
+**Installation**
+Clone the repository (if applicable):
+
+git clone https://github.com/your-username/springoauth2demo.git
+cd springoauth2demo
+Install dependencies and configure your environment:
+
+./mvnw clean install
+
+**Configuration**
+Edit the src/main/resources/application.properties file:
+
+spring.application.name=springoauth2demo
+server.port=3000
+
+auth0.audience=https://your-authaction-api-identifier
+auth0.domain=your-authaction-tenant-domain
```

Comment:
use `authaction.audience` and `authaction.domain` update relevant java code as well.